### DISCUSSION TITLE
Is there a setting to prevent the Twitter banners for downloading?

### DISCUSSION THREAD

**Discussion by Aston77** [AAndyProgram/SCrawler#129] [Q&A] [answered]
I notice if I download a user from Twitter that the profile banner is downloaded as well. If I delete that banner image from file explorer it will download again.

Does one of the settings in the Twitter config allow for components like the banner to download 1x or not be downloaded at all?

**Comment by AAndyProgram** [comment, marked as answer]
Actually, no. You are the first one to mention it. Other users have asked me to add a download of all user media stuff that can be viewed, including the icon and banner. I added this a few releases ago. Is it too annoying?

I think I can add this setting in the next version. But it won't be too soon. I'll be very busy for at least the next couple of weeks.

**Reply by Aston77** [reply]
It's not that it is overly annoying it's more that from what I can tell there is no specific consistent identifier for it. It appears to be treated essentially as any other timeline photo so if the downloaded images are moved out of the directory there is no simple way to filter the banner and Twitter profile picture out of the selection. So they get downloaded again the next time around.

You save the userpicture, profilepicture & profileicon in the same directory, but because they are consistently named its minimal effort to exclude them from a selection.

Just a quirk, nothing that can't be lived with. There are a number of settings for each site and I figured I'd just ask if that was already part of one.

**Reply by AAndyProgram** [reply]
Btw, have you seen [Feed](https://github.com/AAndyProgram/SCrawler/wiki#feed)? This content (banner and icon) is not included in the feed. And it's very convenient to see the last downloaded content, especially with [automation](https://github.com/AAndyProgram/SCrawler/wiki/Settings#automation).

**Reply by AAndyProgram** [reply]
Added. Please update to the [latest release](https://github.com/AAndyProgram/SCrawler/releases/latest).

The [setting](https://github.com/AAndyProgram/SCrawler/wiki/Settings#downloading) is called `Update user icon and banner every time (where supported)` and can be found at: SCrawler (Main window) - Settings -  Downloading

**Reply by Aston77** [reply]
> Added. Please update to the [latest release](https://github.com/AAndyProgram/SCrawler/releases/latest).
> 
> The [setting](https://github.com/AAndyProgram/SCrawler/wiki/Settings#downloading) is called `Update user icon and banner every time (where supported)` and can be found at: SCrawler (Main window) - Settings - Downloading

Sweet!.

BTW - Thanks for getting Twitter fixed.
